en/getting_started/download_and_install.md

```diff
@@ -53,7 +53,7 @@ Before installing *QGroundControl* for the first time:
    ```sh
    sudo usermod -a -G dialout $USER
    sudo apt-get remove modemmanager -y
-   sudo apt install gstreamer1.0-plugins-bad gstreamer1.0-libav gstreamer1.0-gl -y
+   sudo apt install gstreamer1.0-plugins-bad gstreamer1.0-libav gstreamer1.0-gl libxcb-xinerama0 -y
```

Comment:
@DonLakeFlyer This looks reasonable, but is news to me. When did the dependency go in?

Reply:
Only the most recent release (4.2.0) needs the package, the previous stable version (4.1.4) does not.